### ISSUE TITLE
Estimating on training set and validating on test set

### ISSUE THREAD

**Issue by hermandr** [radiant-rstats/radiant.model#7]
Thank you for this great tool, I am able to do analysis without much coding!

I would like to estimate model on train set and validate on test set.

I create a new column in data menu with new column "training"
Then in data/view I set training == 1 as a filter
I go to model and in OLS create a regression model.
So now I want to predict on training == 0 dataset.
Also use the evaluate regression.

How to do with the UI easily?

Thank you

**Comment by vnijs** [comment]
@hermandr The easiest approach is to (1) set a filter, (2) predict for all data, and (3) use Model > Evaluate regression to look at results for training or test data.
